docs/overview/roadmap.md

```diff
@@ -5,23 +5,29 @@ section: Overview
 permalink: /docs/overview/roadmap/
 ---
 
-#Roadmap
+# Roadmap
```

Comment:
I think I found why.

Sometimes when typing `#` to make a heading, authors keep the `alt` key pressed when pressing `space`. `alt+space` = unbreakable space, which obviously will prefix the heading with a visible space.